style/components/bookmark.css

```diff
@@ -0,0 +1,30 @@
+.bookmark {
+  position: absolute;
+  top: -5px;
+  right: 8px;
+  font-size: 35px;
+  color: rgb(250, 137, 7);
+}
+
+.bookmark__btn .bookmark__btn--filled {
+  display: none;
+  cursor: pointer;
```

Comment:
As discussed earlier, you could think about not giving the bookmark a cursor: pointer property; some say it is just for links.